packages/flutter/lib/src/gestures/positioned_details.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
nit: If you really intend to make more gesture detail base classes we probably can rename this file as `gesture_details.dart`.